### PR TITLE
Update crispritz to 1.0.8 (solve problem with OMP library)

### PR DESCRIPTION
**Pull request by samuelecancellieri**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/crispritz/meta.yaml [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 package:
   name: crispritz
   version: {{ version }}
 
 source:
   url: https://github.com/InfOmics/CRISPRitz/archive/v{{ version }}.tar.gz
-  sha256: ec616f6f20c498dc6b84232c52daac4462640fd0e3630b1e267140e95d3e2f63
-
+  sha256: 2030530c40a713e0e387b80b4283bcb6aa12c942ecde58ec45cfd0e5c8285071
+
```
Comment (epruesse) [inline recipes/crispritz/meta.yaml]: trailing space

File: recipes/crispritz/meta.yaml [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 source:
   url: https://github.com/InfOmics/CRISPRitz/archive/v{{ version }}.tar.gz
-  sha256: ec616f6f20c498dc6b84232c52daac4462640fd0e3630b1e267140e95d3e2f63
-
+  sha256: 2030530c40a713e0e387b80b4283bcb6aa12c942ecde58ec45cfd0e5c8285071
+  
 build:
-  number: 0
-  skip: True # [osx]
+  number: 1
```
Comment (epruesse) [inline recipes/crispritz/meta.yaml]: reset to 0 for new version

File: recipes/crispritz/meta.yaml [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
 
 requirements:
   build:
-    - {{compiler('cxx')}} =5.4
+    - {{compiler('cxx')}} =5.4 # [not osx]
```
Comment (epruesse) [inline recipes/crispritz/meta.yaml]: This is very very likely to break in all manner of unexpected ways. The software does not compile with gcc 7.2 at all? Can't be patched?

Comment (samuelecancellieri) [reply recipes/crispritz/meta.yaml]: Hi,
I'm sorry but I tried every version except the 5.4 for gcc and no one can run properly.
I discovered it's a problem with the dynamic bitset variable in the boost library, but I have not found a solution except using the mentioned gcc version.
Another question, I'm having a problem with openmp on the clang compiler for macOS X, I included openmp in the recipe, but the error remains, there's something I'm missing?

Thanks a lot.

**Comment by epruesse** [comment]
Did you give up?

File: recipes/crispritz/meta.yaml (hunk 3 above)
Comment (epruesse) [reply recipes/crispritz/meta.yaml]: I'd have to look at it. Boost should compile fine. It usually is just some warnings that became errors as GCC becomes more stringent. I'd have to compile myself to see. Most of the time, some small patches suffice to fix those compilation issues.

**Comment by samuelecancellieri** [comment]
> Did you give up?

Hi,
I have another work to do, so I can't lose so much time on this one.
I will retry in a few days when I will be more free. 
Don't waste your time on this.

Thanks a lot.

**Pull request closed without merging**